src/components/Tree/Tree.props.ts

```diff
@@ -25,6 +25,15 @@ export type TreeProps = {
    */
   checkable?: boolean,
 
+  /**
+   * (Controlled) Specifies the keys of the checked treeNodes (PS: When this specifies the key of a treeNode
```

Comment:
what do you mean by "Controlled" ?

Reply:
It's a frontend concept that indicate that the component relies on a external state and stops updating and reading its internal one